vnext/Microsoft.ReactNative.Cxx/JSValueReader.h

```diff
@@ -166,10 +145,10 @@ inline void ReadValue(IJSValueReader const &reader, /*out*/ std::string &value)
       value = reader.GetBoolean() ? "true" : "false";
       break;
     case JSValueType::Int64:
-      value = basic_string_convert<char>(reader.GetInt64());
+      value = std::to_string(reader.GetInt64());
       break;
     case JSValueType::Double:
-      value = basic_string_convert<char>(reader.GetDouble());
+      value = std::to_string(reader.GetDouble());
```

Comment:
Please replace it with the following code to make the unit test to pass:
``` C++
{
      std::stringstream ss;
      ss << value;
      value = ss.str();
      break;
}
```

Reply:
It requires use of the curly braces as in the code snippet. I.e.
``` C++
case JSValueType::Double: {
  std::stringstream ss;
  ss << reader.GetDouble();
  value = ss.str();
  break;
}
```

Reply:
I apologize. I didn't see the ```{``` and ```}``` initially.
Just for the reference it is https://stackoverflow.com/a/92439

But even correctly applying the suggestion would not have made my compiler happy:
```
JSValueReader.h:180:13: error: no viable overloaded '='
      value = ss.str();
      ~~~~~ ^ ~~~~~~~~
```